metricbeat/module/docker/cpu/_meta/fields.yml

```diff
@@ -6,31 +6,37 @@
   fields:
     - name: kernel.pct
       type: scaled_float
+      scaling_factor: 1000000
```

Comment:
I remember a scaling factor > 1000 will not have any compression advantage anymore over just using float instead of scaled_float. So I suggest we just change to float here. Also below.

Reply:
Umm, if I understood well, `scaled_float` is stored as an integer, wouldn't it be a breaking change to change it to float? (OTOH changing the `scaling_factor` could also be a breaking change)

Reply:
As each version has a new template, things should not break here. From a query perspective I think it was already like a float before so having a float afterwards should just work.

If someone ingest with the old template he would just have lower accuracy but not get an error.

@jpountz Is the above correct? The ingestion and quering of `float` and `scaled_float` looks identical?

Reply:
Assuming all values are between 0 and 1, this field will likely require only 20 bits per value, which is still better than a float. However I'm very surprised you need so much precision for a percentage, why are 100 or 1000 not enough?

Reply:
@jpountz wdyt? would it be ok to change to `float` for this?

Reply:
The fact it may go over 1 is not an issue, I still think scaled_float is a better fit. However let's use a more reasonable scaling factor like 100?

Reply:
Oh now I wonder that you might have interpreted the scaling factor as an upper bound of the values, while it's not. It just means that Elasticsearch will store `x * scaling_factor` as a `long` internally.

Reply:
Oh, ok, understood now :) Then 100/1000 will be probably enough, yes. Thanks @jpountz !!

I'll leave it empty as it was before, default seems to be 1000.